### PR TITLE
Implemented database connection for issues

### PR DESCRIPTION
**Pull request by litityum**
Implemented database connection for issues and implemented helper methods for inserting or selecting issues

### CODE REVIEW

File: practice-app/src/main/helpers/issue_helper.py [hunk 1]
```diff
@@ -29,3 +35,111 @@ def get_issue(json_issue):
...
+        cur.execute(
+            "INSERT OR IGNORE INTO Assignees(issue_number, assignee) VALUES (?,?)",
+            (issue.number, assignee))
+    for label in issue.labels:
+        cur.execute(
+            "INSERT OR IGNORE INTO Labels(issue_number, label) VALUES (?,?)",
+            (issue.number, label))
+
+
+def insert_single_issue(issue: Issue):
```
Comment (veyisTurgut) [inline practice-app/src/main/helpers/issue_helper.py:57]: maybe you can delete this function and send single issue as `list(issue)` to `insert_multiple_issue()`. But just a suggestion, not necessary.

File: practice-app/src/main/helpers/issue_helper.py [hunk 2]
```diff
@@ -29,3 +35,111 @@ def get_issue(json_issue):
...
+        return data.fetchone()[0]
+    except Exception:
+        print(f'An exception occurred while getting issue count')
+        return -1
+    finally:
+        con.commit()
+        con.close()
+
+
+def get_issue(issue_number: int) -> Issue:
```
Comment (veyisTurgut) [inline practice-app/src/main/helpers/issue_helper.py:96]: same comment as above

**Review by veyisTurgut — APPROVED** [review]
I am impressed!

File: practice-app/src/main/helpers/issue_helper.py (hunk 1 above)
Comment (litityum) [reply practice-app/src/main/helpers/issue_helper.py:57]: It makes so much sense

**Pull request merged**